pygmt/tests/test_text.py

```diff
@@ -275,8 +275,7 @@ def test_text_justify_parsed_from_textfile():
     Print text justified based on a column from textfile, using justify=True
     boolean operation.
 
-    Loosely based on "All great-circle paths lead to Rome"
-    gallery example at
+    Loosely based on "All great-circle paths lead to Rome" gallery example at
     https://gmt.soest.hawaii.edu/doc/latest/gallery/ex23.html
```

Comment:
Let's use the new link instead:
```suggestion
    https://docs.generic-mapping-tools.org/latest/gallery/ex23.html
```